proto/sdk/blocksdk/v1/tx.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+package sdk.blocksdk.v1;
+
+option go_package = "github.com/skip-mev/block-sdk/x/blocksdk/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/msg/v1/msg.proto";
+import "cosmos_proto/cosmos.proto";
+import "google/api/annotations.proto";
+import "amino/amino.proto";
+import "sdk/blocksdk/v1/blocksdk.proto";
+
+// Msg defines the x/blocksdk Msg service.
+service Msg {
+  option (cosmos.msg.v1.service) = true;
+
+  // UpdateLane defines a method to update an existing lane in the store.
+  rpc UpdateLane(MsgUpdateLane) returns (MsgUpdateLaneResponse) {
+    option (google.api.http).post = "/block-sdk/blocksdk/v1/update_lane";
```

Comment:
Not rly sure what this tag will do? Given that the MsgService itself is not exposed via a REST / http API? Is this necessary